content/en/monitors/incident_management.md

```diff
@@ -7,8 +7,6 @@ description: Create and manage incidents
 
 {{< img src="monitors/incidents/incidents-top-1.png" alt="Incident Management"  style="width:80%;">}}
 
-## Overview
-
 Any event that may lead to a disruption in your organization’s services can be described as an incident, and it is often necessary to have a set framework for handling these events. Datadog’s Incident Management feature provides a system through which your organization can effectively identify and mitigate incidents. 
 
 Incidents live in Datadog alongside the metrics, traces, and logs you are collecting. You can view and filter incidents that are relevant to you.
```

Comment:
Incidents and outages can be managed in Datadog alongside the metrics, traces, logs, and other data you are collecting. You can view and filter incidents that are relevant to you and your team.